float64 stamp
float32 minProbability
float32 maxProbability
float32 period
bool set_color
float32 red
float32 green 
float32 blue 
float32 alpha 
int32 type 
string name
int32 roi
string filename
---
int32 number
